applications_tests/gls_sharp_navier_stokes_2d/moving_ib_gls.prm

```diff
@@ -18,6 +18,9 @@ end
 #---------------------------------------------------
 subsection physical properties
     set kinematic viscosity            = 0.1
+    subsection gravity
+        set Function expression =0;-9.81;0
```

Comment:
I'm not sure, but why do you need to make gravity different, it's the one you apply on the particles?